dev.py

```diff
@@ -72,7 +72,20 @@ def cmd_install_browsers(args):
 
 def cmd_all_run_e2e(args):
     """"Run e2e tests for all examples and templates"""
-    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
+    for project_dir in TEMPLATE_DIRECTORIES:
+        e2e_dir = next(project_dir.glob("**/e2e/"), None)
+        if e2e_dir:
+            with tempfile.TemporaryDirectory() as tmp_dir:
+                run_verbose(['python', '-m', 'venv', f"{tmp_dir}/venv"])
+                wheel_files = list(project_dir.glob("dist/*.whl"))
+                if wheel_files:
+                    wheel_file = wheel_files[0]
+                    run_verbose([f"{tmp_dir}/venv/bin/pip", "install", f"{str(wheel_file)}[devel]"], cwd=str(project_dir))
```

Comment:
I think that this install step with creating venv at first could be in `cmd_all_install_wheel_packages`.
Here could be only running pytest (within two loops of course). 
Also, I guess that the `cmd_all_install_python_deps` function could be removed completely.
What do you think?

Reply:
I like to have separate commands to quickly setup dev env, not only to run e2e tests.